packages/widget/src/controllers/transfers/fungible-token-transfer.ts

```diff
@@ -35,6 +36,7 @@ export class FungibleTokenTransferController implements ReactiveController {
   public waitingTxExecution: boolean = false;
   public transferTransactionId?: string;
   public errorMessage: string | null = null;
+  public invalidDestinationAddressErrorMessage: string | null = null;
```

Comment:
since controller doesn't really care about address. Maybe we could just have null state for desinationAddress:
```
empty string = missing address
null/undefined = invalid address
anything else valid address
```
wdyt?

Reply:
@mpetrunic In general, I like the idea as I also would like to avoid sending the state back to the parent. The only issue I see is that if I enter the substrate address when the EVM will be expected, I will have an error on the field level, but the button will have an "enabled" state, which is incorrect. So I will call "validateAddress" one more time. I pushed the code with the changes